src/main/java/org/opensearch/knn/common/KNNConstants.java

```diff
@@ -51,4 +51,9 @@
     public static final String COMPOUND_EXTENSION = "c";
 
     public static final String JNI_LIBRARY_NAME = "OpensearchKNN";
+
+    public static final String MODEL_BLOB_PARAMETER = "model_blob";
+
+    public static final String MODEL_INDEX_MAPPING_PATH = "mappings/model-index.json";
+    public static final String MODEL_INDEX_NAME = ".knn-model-index";
```

Comment:
Should the file name start with opensearch? May be confirm other plugins system indices naming convention with openSearch changes.

Reply:
In [Conventions](https://github.com/opensearch-project/opensearch-plugins/blob/main/CONVENTIONS.md#indices), the only requirement is prefixed with ".".

1. [AD](https://github.com/opensearch-project/anomaly-detection/blob/main/src/main/java/org/opensearch/ad/constant/CommonName.java) is still using "opendistro-"
2. [ISM](https://github.com/opensearch-project/index-management/blob/main/src/main/kotlin/org/opensearch/indexmanagement/IndexManagementPlugin.kt#L189) also prefixes with ".opendistro".

That being said, I think ".opensearch-knn-model-index" is good. I will update.